backend/de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/ddl/functions/update_TRL_Tables_On_AD_Element_TRL_Update.sql

```diff
@@ -1,114 +1,62 @@
-DROP FUNCTION IF EXISTS update_trl_tables_on_ad_element_trl_update(numeric, character varying);
+DROP FUNCTION IF EXISTS update_trl_tables_on_ad_element_trl_update(
+    p_AD_Element_ID numeric,
+    p_AD_Language   character varying
+)
+;
 
-CREATE OR REPLACE FUNCTION update_trl_tables_on_ad_element_trl_update
-(
-	p_AD_Element_ID numeric,
-	p_AD_Language character varying = null
+CREATE OR REPLACE FUNCTION update_trl_tables_on_ad_element_trl_update(
+    p_AD_Element_ID numeric = NULL,
+    p_AD_Language   character varying = NULL
 )
-RETURNS void
+    RETURNS void
 AS
 $BODY$
-DECLARE
-	update_count integer;
-BEGIN
-	--
-	-- AD_Column_Trl
-	PERFORM update_Column_Translation_From_AD_Element(p_AD_Element_ID, p_AD_Language);
 
-	--
-	-- AD_Field_TRL
-	PERFORM update_FieldTranslation_From_AD_Name_Element(p_AD_Element_ID, p_AD_Language);
+BEGIN
+    --
+    -- AD_Column_Trl
```

Comment:
IMHO you don't need those `-- AD_Column_Trl`, `-- AD_Field_Trl`, etc comments.
The function names are self-explanatory.